app/javascript/pages/browse/MapBrowser.vue

```diff
@@ -2,105 +2,131 @@
   <section class="MapBrowser">
     <!-- FIXME Pull access-token via environment variable -->
     <mapbox
-        access-token="pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"
-        :map-options="{
-          style: 'mapbox://styles/mapbox/streets-v11',
-          center: [-75.1635262, 39.9527237],
-          zoom: 9,
-        }"
-        :geolocate-control="{
-          show: true,
-          position: 'top-left',
-        }"
-        @map-load="loaded"
-        @map-zoomend="zoomend"
-        @map-click:points="clicked"
-        @geolocate-error="geolocateError"
-        @geolocate-geolocate="geolocate"
-      />
+      :access-token= this.accessToken
+      :map-options="{
+        style: 'mapbox://styles/mapbox/streets-v11',
+        center: [-96, 37.8],
+        zoom: 4,
+      }"
+      :geolocate-control="{
+        show: true,
+        position: 'top-left',
+      }"
+      @map-load="loaded"
+      @map-zoomend="zoomend"
+      @map-click:points="clicked"
+      @geolocate-error="geolocateError"
+      @geolocate-geolocate="geolocate"
+    />
   </section>
 </template>
 
 <script>
-import Mapbox from 'mapbox-gl-vue'
+  import Mapbox from 'mapbox-gl-vue'
+  const mbxClient = require('@mapbox/mapbox-sdk');
+  const geoCoding = require('@mapbox/mapbox-sdk/services/geocoding');
+  //  FIXME Attempt to pass accessToken in script tag as seen in line 6
+  const baseClient = mbxClient({ accessToken: "pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"});
+  const geoCodingService = geoCoding(baseClient)
 
-export default {
-  components: { Mapbox },
-  methods: {
-    loaded(map) {
-      var geojson = {
-        'type': 'FeatureCollection',
-        'features': [
-          {
-            'type': 'Feature',
-            'properties': {
-              'message': 'Foo',
-              'icon': 'harbor',
-              'iconSize': [60, 60]
-            },
-            'geometry': {
-              'type': 'Point',
-              'coordinates': [-75.1635262, 39.9527237]
-            }
-          },
-        ]
-      }
+  export default {
+    props: {
+      accessToken: String,
+      contributions: {type: Array, default: () => []},
+      filters: Object,
+    },
+    components: { Mapbox },
+    methods: {
+      loaded(map) {
```

Comment:
This looks great. On a second pass in the future, it could be worth taking a pass at extracting some of the sdk functions out into a separate module. There's a bit going on in this `loaded` method :).

Reply:
`this.` is necessary because we are accessing MapBrowse's props and `{{ accessToken }}` didn't pass the token at all.